src/main/java/org/benetech/servicenet/repository/GeocodingResultRepository.java

```diff
@@ -37,4 +39,23 @@
     @Override
     @CacheEvict(value = "addressCache", allEntries = true)
     void deleteAllInBatch();
+
+    @Query("SELECT DISTINCT gr.postalCode FROM GeocodingResult gr "
+        + "LEFT JOIN Location l ON gr in elements(l.geocodingResults) "
```

Comment:
why it's done like that and not select form location and join geocoding? is it have better performance?